packages/performance/src/services/transport_service.ts

```diff
@@ -112,7 +112,7 @@ function dispatchQueueEvents(): void {
   };
   /* eslint-enable camelcase */
 
-  postToEndpoint(data, staged).catch(() => {
+  sendEventsToFl(data, staged).catch(() => {
```

Comment:
NIT: One thing we can do is to unwrap `sendEventsToFl` in the current function, since it doesn't require endpoint switching any more.

Reply:
Unwrapping is a good idea. But that makes the function look too big. So I would prefer it to be as is to make it easy to understand.